webroot/js/debug_kit.js

```diff
@@ -1,25 +1,29 @@
-var baseUrl, toolbar;
+import './jquery.js';
+import {Toolbar} from './toolbar-app.js';
 
-var elem = document.getElementById('__debug_kit_app');
-if (elem) {
-  window.__debugKitId = elem.getAttribute('data-id');
-  window.__debugKitBaseUrl = elem.getAttribute('data-url');
-  window.__debugKitWebroot = elem.getAttribute('data-webroot');
-  elem = null;
-}
+( function( $ ) {
+    'use strict';
 
-$(document).ready(function() {
-  toolbar = new Toolbar({
-    toolbar: $('#toolbar'),
-    content: $('#panel-content-container'),
-    panelButtons: $('.panel'),
-    panelClose: $('#panel-close'),
-    keyboardScope : $(document),
-    currentRequest: __debugKitId,
-    originalRequest: __debugKitId,
-    baseUrl: __debugKitBaseUrl,
-    webroot: __debugKitWebroot,
-  });
+    let elem = document.getElementById( '__debug_kit_app' );
+    if( elem ) {
```

Comment:
The spacing in these blocks isn't consistent with the rest of the code, can we make it all formatted the same way? Preferably using a linter/formatter so that we can ensure consistency in the future as well.